src/panel/download_card/DownloadCardContainer.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+// const PropTypes = React.PropTypes; TODO
+import {observer} from "mobx-react";
+import MainStore from "../common/MainStore";
+import downloadActions from "./downloadActions";
+import DownloadCard from "./DownloadCard";
+import {Collapse} from "@blueprintjs/core";
+
+@observer
+class DownloadCardContainer extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			title: "",
+			id: ""
+		};

```

Comment:
Let's use MobX to manage local state (like in SpoilerCardContainer) rather than `this.setState`. More details in CONTRIBUTING.md